sdk/forms/src/main/java/com/klaviyo/forms/InAppForms.kt

```diff
@@ -17,8 +17,16 @@ import com.klaviyo.forms.webview.KlaviyoWebViewClient
 import com.klaviyo.forms.webview.WebViewClient
 
 /**
- * Load in-app forms data and display a form to the user if applicable based on the forms
- * configured in your Klaviyo account. Note [Klaviyo.initialize] must be called first
+ * Start an In-App Forms session.
+ *
+ * This will load forms data and establish ongoing listeners to present a form to the user
+ * whenever a form is triggered by an event or condition according to the targeting and behavior
+ * settings configured for forms in your Klaviyo account.
+ *
+ * Note: a public API key is required, so [Klaviyo.initialize] must be called first.
+ * If the API key changes, the session will be re-initialized automatically with the new key.
```

Comment:
@ab1470 on Ajay's advice, I took some of the deeper behavioral details out of the README, leaving that to focus on integration steps, and elaborated instead in some code comments on public methods.